include/oneapi/dpl/numeric

```diff
@@ -30,6 +30,7 @@
 
 // ATTENTION!!! Include the header from the C++ standard library before the oneDPL config header to see whether the _PSTL_VERSION macro is defined
 #include <numeric>
+#include "oneapi/dpl/internal/gcd_impl.h"
```

Comment:
Why this header is placed before `onedpl_config.h`, not after?

Reply:
I placed these headers by alphabetical order before, now I have arranged onedpl_config.h  at front position.